css/resize/demos/resize-playground/index.html

```diff
@@ -0,0 +1,141 @@
+<!DOCTYPE html>
+<html lang="ru">
+<head>
+  <title>Resize Playground — Дока</title>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500&display=swap">
+  <style>
+    *, *::before, *::after {
+      margin: 0;
+      padding: 0;
+      box-sizing: border-box;
+    }
+
+    body {
+      min-height: 100vh;
+      padding: 50px;
+      display: flex;
+      align-items: center;
+      justify-content: center;
+      background-color: #18191c;
+      color: #FFFFFF;
+      font-family: "Roboto", sans-serif;
+      font-size: 18px;
+    }
+
+    .resize-demo {
+      display: grid;
+      grid-template-columns: auto minmax(0, 1fr);
+      gap: 32px;
+
+      width: 100%;
+      max-width: 1360px;
+    }
+
+    .resize-demo__sidebar {
+      display: flex;
+      flex-direction: column;
+    }
+
+    .resize-demo__values {
+      padding: 16px;
+    }
+
+    .resize-demo__value:hover,
+    .resize-demo__value-label:hover {
+      cursor: pointer;
+    }
+
+    .resize-demo__item + .resize-demo__item {
+      margin-top: 12px;
+    }
+
+    .resize-demo__legend {
+      padding: 0 8px;
+    }
+
+    .resize-demo__content {
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        
+        width: 100%;
+        max-width: 100%;
+        
+        height: 100%;
+        max-height: 50vh;
+
+        padding: 8px;
+
+        background-color: teal;
+        overflow: auto;
+
+        resize: both;
+    }
```

Comment:
```suggestion
    .resize-demo__content {
        display: flex;
        justify-content: center;
        align-items: center;

        width: 100%;
        max-width: 100%;

        height: 100%;
        max-height: 50vh;

        padding: 8px;

        background-color: #2E9AFF;
        color: #000000;
        overflow: auto;

        resize: both;
    }
```

Подставила фирменный цвет раздела CSS и покрасила текст в чёрный, чтобы сохранить читаемость.